srcalc/src/test/java/gov/va/med/srcalc/web/view/VariableEntryTest.java

```diff
@@ -67,4 +69,31 @@ public final void testRetrievedCardiacAge()
     	
     	assertEquals(expected, entry.getDynamicValues());
     }
+    
+    @Test
+    public final void testWithRetrievedLabs()
+    {
+        final List<AbstractVariable> vars = SampleModels.sampleVariableList();
+        final Patient patient = SampleCalculations.dummyPatientWithLabs(1);
+        final VariableEntry entry = VariableEntry.withRetrievedValues(vars, patient);
+        final HashMap<String, String> expected = new HashMap<>();
+        
+        expected.put(vars.get(1).getKey(), String.valueOf(patient.getAge()));
+        expected.put(vars.get(2).getKey(), patient.getGender());
+        expected.put(vars.get(5).getKey(), VariableEntry.SPECIAL_NUMERICAL);
+        
+        final String numericalString = VariableEntry.SEPARATOR + VariableEntry.SPECIAL_NUMERICAL;
+        final String dateString = numericalString + VariableEntry.SEPARATOR + VariableEntry.RETRIEVAL_STRING;
```

Comment:
Bleh these strings are such tight coupling to an implementation detail of `VariableEntry`. I know I started the trend when I wrote the class, but I think we could (and should) hide these string constants behind a public interface in `VariableEntry`. Let's talk about it.